packages/env/lib/build-docker-compose-config.js

```diff
@@ -1,13 +1,28 @@
 'use strict';
+
 /**
- * External dependencies
+ * @typedef {import('./config').Config} Config
+ * @typedef {import('./config').Source} Source
  */
-const fs = require( 'fs' );
-const path = require( 'path' );
```

Comment:
Maybe a bad rebase? Removing these imports is causing some errors.

Reply:
yep! when I first started this PR, we were able to remove these deps, but in the meantime we added some other functionality which uses them :D